File: java/v1/src/main/java/com/api/v1/borrower/services/UpdateBorrowerService.java
package com.api.v1.borrower.services;

import com.api.v1.borrower.domain.Borrower;
import com.api.v1.borrower.dtos.UpdateBorrowerRequestDto;

import reactor.core.publisher.Mono;

public interface UpdateBorrowerService {

    Mono<Borrower> update(String ssn, UpdateBorrowerRequestDto request);

}

File: koltlin/v1/src/main/kotlin/com/api/v1/borrower/services/UpdateBorrowerService.kt
package com.api.v1.borrower.services

import com.api.v1.borrower.domain.Borrower
import com.api.v1.borrower.helpers.UpdateBorrowerRequestDto
import reactor.core.publisher.Mono

interface UpdateBorrowerService {

    fun update(ssn: String, request: UpdateBorrowerRequestDto): Mono<Borrower>;

}